pkg/cnb/remote_store_reader_test.go

```diff
@@ -77,11 +96,19 @@ func testRemoteStoreReader(t *testing.T, when spec.G, it spec.S) {
 			})
 			require.NoError(t, err)
 
-			fakeClient.AddImage(buildpackageA, buildPackageA, "", expectedKeychain)
+			fakeClient.AddImage(buildpackageA, buildPackageA, expectedKeychain)
```

Comment:
this was not introduced by you, but can we change the reponame and the image name because this is really difficult to read, it being `buildpackageA` and `buildPackageA`

Reply:
Same thing with `buildpackageB` and `buildPackageB`

Reply:
Agreed